services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_organization_spec.rb

```diff
@@ -404,6 +404,24 @@
           expect(described_class.new(district).data[:traits][:active_students_all_time]).to eq(2)
         end
       end
+
+      context 'excludes classrooms where teacher is coteacher' do
```

Comment:
Again, take or leave this, but I'd just override the `let!` call instead of using a `before` block:
```
let!(:classroom_teacher1) { create(:classrooms_teacher, user: teacher2, classroom: classroom1, role: 'coteacher') }
```